app/models/activity.rb

```diff
@@ -1,5 +1,7 @@
 class Activity < ActiveRecord::Base
   has_and_belongs_to_many :users
   has_one :creator, class_name: User
+  validates_presence_of :name
```

Comment:
@thutterer  it is really better to use new format of validations
like 

validates <column_name_symbol>, presence: true